src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf

```diff
@@ -100,7 +100,7 @@
             </trans-unit>
             <trans-unit id="25">
                 <source>This value is not valid.</source>
-                <target>Deze waarde is ongeldig.</target>
+                <target>Deze waarde is niet geldig.</target>
```

Comment:
Point is i think these are good _default_ translations. If you want a certain tone, syntax, whatsoever you are probably using custom translations anyway (by using new translations keys or overriding the default ones).